src/cmdc_tools/datasets/official/WA/__init__.py

```diff
@@ -50,13 +56,58 @@
     "Yakima",
 ]
 
+
 DASHBOARD_URL = (
+    "https://coronavirus.wa.gov/what-you-need-know/covid-19-risk-assessment-dashboard"
+)
+
+POWER_BI_DASHBOARD_ENDPOINT = (
     "https://wabi-us-gov-virginia-api.analysis.usgovcloudapi.net"
     "/public/reports/querydata?synchronous=true"
 )
 
 
-def run_query_for_county(county):
+@dataclasses.dataclass
+class PowerBIQueryDetails:
+    """IDs needed to query latest version of Power BI objects."""
+    resource_key: str
+    model_id: int
+    database_id: str
+    report_id: str
+
+
+def _get_resource_key(wa_dashboard_url):
```

Comment:
Let's add a `:str` type here

We (me and @cc7768 )  haven't been great examples of this, but are currently going through a revolution of sorts to add this everywhere to make it easier for maintenance and onboarding new team members.